modules/apigee_edge_teams/src/Entity/Form/TeamForm.php

```diff
@@ -159,6 +159,9 @@ public function buildEntity(array $form, FormStateInterface $form_state) {
     /** @var \Drupal\apigee_edge_teams\Entity\TeamInterface $team */
     $team = parent::buildEntity($form, $form_state);
 
+    if (NULL === $this->orgController) {
+      $this->orgController = \Drupal::service('apigee_edge.controller.organization');
```

Comment:
This fixes the null problem, but how this can be NULL when it is DI-d above?

Reply:
This issue is coming in 3x only, when added new Team field with multi-value field setting.

Reply:
I saw the report, but by reading the code in this class I still do not see why/how this can be NULL, when the service is injected above. 
Depending on `\Drupal::service(()` is classes only acceptable as a BC bridge that gets removed in the next major version. Is this going to be removed or have to stay?

Reply:
This issue was caused by `$orgController` private access specifier.
Resolved

Reply:
Ahh so the actual issue was related to https://www.drupal.org/project/drupal/issues/3110266